forum/constants.py

```diff
@@ -0,0 +1,4 @@
+"""Constants for forum app."""
+
+RETIRED_TITLE = "[deleted]"
+RETIRED_BODY = "[deleted]"
```

Comment:
may be we can use one variable instead of two as the value is same.

Reply:
tried to keep it consistent with `cs_comments_service`